docker-compose.dev.yml

```diff
@@ -0,0 +1,111 @@
+version: '3.5'
+# Use this script for running up nightfall_3 in 'developer' mode with local
+# bindings.  See the readme for more information.
+services:
+  client1:
+    build:
+      dockerfile: Dockerfile
+      context: ./nightfall-client
```

Comment:
These files were removed in this earlier [PR](https://github.com/EYBlockchain/nightfall_3/pull/387).
The context might need to be changed to root and the dockerfiles from root too